### PR TITLE
Splitter: update focusable element in MVC and .Net Demo

### PR DESCRIPTION
**Pull request by nikkithelegendarypokemonster**

### CODE REVIEW

File: MVCDemos/Views/Splitter/Overview.cshtml [hunk 1]
```diff
@@ -6,8 +6,8 @@
                 items.Add()
                     .Resizable(true)
                     .Size("140px")
-                    .MinSize("70px")
-                    .Template(new JS("function(data){return paneContentTemplate(data, 'Left Pane')}"));
+                    .MinSize("70px")
```
Comment (EugeniyKiyashko) [inline MVCDemos/Views/Splitter/Overview.cshtml]: Excess spaces

File: MVCDemos/Views/Splitter/Overview.cshtml [hunk 2]
```diff
@@ -66,15 +66,16 @@
...
     }
 
-    function paneContentTemplate(data, paneName) {
+    function paneContentTemplate(data, paneName, element) {
         const dimensionOptions = new Set(['size', 'minSize', 'maxSize']);
         const filteredData = Object.fromEntries(
             Object.entries(data)
                 .filter(([key]) => dimensionOptions.has(key))
         );
+        element.eq(0).attr('tabIndex', '0');
```
Comment (ksercs) [inline MVCDemos/Views/Splitter/Overview.cshtml]: do we really need `eq(0)` here?

File: NetCoreDemos/Views/Splitter/Overview.cshtml [hunk 3]
```diff
@@ -2,56 +2,61 @@
...
-                                        .MinSize("5%")
-                                        .Template(new JS("function(data){return paneContentTemplate(data, 'Nested Left Pane')}"));
+                        .Collapsible(true)
+                        .Splitter(horizontalSplitter =>
+                        {
+                            horizontalSplitter.Orientation(Orientation.Horizontal);
+                            horizontalSplitter.Items(horizontalItems =>
+                            {
+                                horizontalItems.Add()
+                    .Resizable(true)
```
Comment (ksercs) [inline NetCoreDemos/Views/Splitter/Overview.cshtml]: I see a lot if incorrect indents in this file. Fix it please

File: MVCDemos/Views/Splitter/Overview.cshtml [hunk 4]
```diff
@@ -16,7 +16,7 @@
                             .Resizable(true)
                             .Collapsible(true)
                             .MaxSize("75%")
-                            .Template(new JS("function(data){return paneContentTemplate(data, 'Central Pane')}"));
+                            .Template(new JS("function(data, itemIndex, itemElement){return paneContentTemplate(data, 'Central Pane', itemElement)}"));
```
Comment (EugeniyKiyashko) [inline MVCDemos/Views/Splitter/Overview.cshtml]: In jQuery you use the following arguments order 
image
Why do you use other order here?

File: NetCoreDemos/Views/Splitter/Overview.cshtml [hunk 5]
```diff
@@ -66,25 +71,26 @@
...
+        element.eq(0).attr('tabIndex', '0');
 
         return `
-            <div class='pane-content' tabindex="0">
-                <div class='pane-title'>
-                    ${paneName}
-                </div>
-                <div class='pane-state'>
-                    ${getStateText(data)}
+                <div class='pane-content'>
```
Comment (EugeniyKiyashko) [inline NetCoreDemos/Views/Splitter/Overview.cshtml]: There are no any styles for borders. Should they be presented in this PR?

File: MVCDemos/Views/Splitter/Overview.cshtml (hunk 4 above)
Comment (nikkithelegendarypokemonster) [reply MVCDemos/Views/Splitter/Overview.cshtml]: ok i will change the order

File: NetCoreDemos/Views/Splitter/Overview.cshtml (hunk 5 above)
Comment (nikkithelegendarypokemonster) [reply NetCoreDemos/Views/Splitter/Overview.cshtml]: [f3b3d38](https://github.com/DevExpress/devextreme-demos/pull/3206/commits/f3b3d380de5365140a538df371a86eccaa4da34c)

File: MVCDemos/Views/Splitter/Overview.cshtml (hunk 4 above)
Comment (nikkithelegendarypokemonster) [reply MVCDemos/Views/Splitter/Overview.cshtml]: [28ba54b](https://github.com/DevExpress/devextreme-demos/pull/3206/commits/28ba54bc841c98cb4dae42826a2ad9274c6c4ad8)

File: NetCoreDemos/Views/Splitter/Overview.cshtml (hunk 3 above)
Comment (nikkithelegendarypokemonster) [reply NetCoreDemos/Views/Splitter/Overview.cshtml]: [28ba54b](https://github.com/DevExpress/devextreme-demos/pull/3206/commits/28ba54bc841c98cb4dae42826a2ad9274c6c4ad8)

File: MVCDemos/Views/Splitter/Overview.cshtml (hunk 2 above)
Comment (nikkithelegendarypokemonster) [reply MVCDemos/Views/Splitter/Overview.cshtml]: as i have tested we can remove the eq(0)
[28ba54b](https://github.com/DevExpress/devextreme-demos/pull/3206/commits/28ba54bc841c98cb4dae42826a2ad9274c6c4ad8)

File: MVCDemos/Views/Splitter/Overview.cshtml (hunk 1 above)
Comment (nikkithelegendarypokemonster) [reply MVCDemos/Views/Splitter/Overview.cshtml]: [28ba54b](https://github.com/DevExpress/devextreme-demos/pull/3206/commits/28ba54bc841c98cb4dae42826a2ad9274c6c4ad8)

**Review by EugeniyKiyashko — DISMISSED** [review]

File: NetCoreDemos/Views/Splitter/Overview.cshtml [hunk 6]
```diff
@@ -66,25 +71,26 @@
...
             Object.entries(data)
                 .filter(([key]) => dimensionOptions.has(key))
         );
+        element.attr('tabIndex', '0');
 
         return `
-            <div class='pane-content' tabindex="0">
-                <div class='pane-title'>
-                    ${paneName}
+                <div class='pane-content'>
```
Comment (Zedwag) [inline NetCoreDemos/Views/Splitter/Overview.cshtml]: formatting looks broken here (probably excess tab)

File: NetCoreDemos/Views/Splitter/Overview.cshtml [hunk 7]
```diff
@@ -2,44 +2,49 @@
...
 
                         verticalItems.Add()
                             .Collapsible(true)
-                            .Splitter(horizontalSplitter => {
+                            .Splitter(horizontalSplitter =>
+                            {
                                 horizontalSplitter.Orientation(Orientation.Horizontal);
-                                horizontalSplitter.Items(horizontalItems => {
+                                horizontalSplitter.Items(horizontalItems =>
+                                {
```
Comment (Zedwag) [inline NetCoreDemos/Views/Splitter/Overview.cshtml]: looks like a bad formatting, lets revert this

Comment (ksercs) [reply NetCoreDemos/Views/Splitter/Overview.cshtml]: same above

Comment (Zedwag) [reply NetCoreDemos/Views/Splitter/Overview.cshtml]: in several places, right

Comment (nikkithelegendarypokemonster) [reply NetCoreDemos/Views/Splitter/Overview.cshtml]: [c0c934e](https://github.com/DevExpress/devextreme-demos/pull/3206/commits/c0c934ef05f6d28fb12291c26e9e5faad9f8aa6d)

File: NetCoreDemos/Views/Splitter/Overview.cshtml (hunk 6 above)
Comment (nikkithelegendarypokemonster) [reply NetCoreDemos/Views/Splitter/Overview.cshtml]: [c0c934e](https://github.com/DevExpress/devextreme-demos/pull/3206/commits/c0c934ef05f6d28fb12291c26e9e5faad9f8aa6d)

File: MVCDemos/Views/Splitter/Overview.cshtml [hunk 8]
```diff
@@ -6,17 +6,17 @@
                 items.Add()
                     .Resizable(true)
                     .Size("140px")
-                    .MinSize("70px")
-                    .Template(new JS("function(data){return paneContentTemplate(data, 'Left Pane')}"));
-
+                    .MinSize("70px")
```
Comment (ksercs) [inline MVCDemos/Views/Splitter/Overview.cshtml]: trailing spaces here and below

Comment (nikkithelegendarypokemonster) [reply MVCDemos/Views/Splitter/Overview.cshtml]: [a1e2fd2](https://github.com/DevExpress/devextreme-demos/pull/3206/commits/a1e2fd267994e161692a4610db7fe46534ba8c28)

already double checked also in .Net demo trailing spaces have been removed

**Review by ksercs — APPROVED** [review]

**Review by Zedwag — APPROVED** [review]

**Pull request merged**